core/src/main/java/me/mykindos/betterpvp/core/command/Command.java

```diff
@@ -24,6 +24,11 @@ public abstract class Command implements ICommand {
     protected List<String> aliases;
     protected List<ICommand> subCommands;
 
+    public final static List<String> LOG_TYPES = List.of(
+            "Oak", "Birch", "Dark_Oak", "Jungle", "Mangrove",
+            "Acacia", "Spruce", "All"
+    );
+
```

Comment:
Should not go here, you can override the procesTabComplete in individual commands, e.g. CustomEffectSubCommand.java